helm/templates/deployment.yaml

```diff
@@ -75,6 +75,8 @@ spec:
         - name: LOG_REQUESTS
           value: {{ .Values.env.logRequests | squote }}
         {{ end -}}
+        - name: P2P_CONNECTION_MAX_CONNECTIONS
```

Comment:
Is this a magic env var that libp2p checks? how does this new env var get applied?

Reply:
oh i see
https://github.com/elastic-ipfs/bitswap-peer/blob/07d0c9caad10f862abd657e4674b65604ba0faa2/src/config.js#L53